.github/CODEOWNERS

```diff
@@ -4,5 +4,6 @@
 
 ## Docs
 
-/docs/ @DataDog/documentation @DataDog/rum-mobile
+doc/ @DataDog/documentation @DataDog/rum-mobile
+docs/ @DataDog/documentation @DataDog/rum-mobile
```

Comment:
we are not sure yet what the folder name will be?

Reply:
It will be `doc` by Flutter convention, but I think there might be some legacy hanging around? I kept both in to be safe.